src/devices/xiaomi.ts

```diff
@@ -2524,14 +2602,41 @@ const definitions: Definition[] = [
         model: 'LLKZMK12LM',
         vendor: 'Xiaomi',
         description: 'Aqara dual relay module T2',
-        fromZigbee: [fz.on_off, fz.aqara_opple, fz.xiaomi_power],
-        toZigbee: [tz.on_off],
-        meta: {multiEndpoint: true},
+        fromZigbee: [fz.on_off, fz.aqara_opple, fz.xiaomi_power, fzLocal.attributes, fzLocal.attributes_multistate],
+        toZigbee: [tz.on_off, tzLocal.attributes],
+        meta: {
+            multiEndpoint: true,
+            attributes: {
+                'switch_type': [1, 'aqaraOpple', 0x000a, 0x20, {'toggle': 1, 'momentary': 2, 'none': 3}],
```

Comment:
There are already converters available for this which are used by multiple xiaomi devices, no need to add new ones:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/085c42e5da15e28a60293182fed56e72577ec041/src/converters/toZigbee.js#L2267